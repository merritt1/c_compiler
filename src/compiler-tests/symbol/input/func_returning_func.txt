int myFunc(void);
int myFunc(void) {
    return;
}
int main(void) {
    return myFunc;
}
