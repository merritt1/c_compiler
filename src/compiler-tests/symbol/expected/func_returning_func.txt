


***ERROR: Function may not return an array or function.


=============== PARSE TREE ===============
signed int (myFunc(void));
signed int (myFunc(void)) {
return;
}
signed int (main(void)) {
return myFunc;
}
================= SYMBOLS ================
Symbol Table ID: 0
  myFunc -> function(signed int, [void])
  main -> function(signed int, [void])

